src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/BaseNodeModel.tsx

```diff
@@ -6,12 +6,15 @@ import { StandardIncomingPortModel, StandardOutgoingPortModel } from '../nodes/P
 export class BaseNodeModel extends NodeModel {
   public isStartNode = false
   public isHighlighted = false
+  public isResizable = false
```

Comment:
Ho can I make this turn to true as a user ?  I really didn't find out by clicking all around